src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,48 @@
+import { v4 as uuid } from 'uuid';
+import cn from 'classnames';
+
+import { NO_PARENT_INFO } from '../../constants/constants';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+  selected: boolean;
+};
+
+export const PersonItem: React.FC<Props> = ({ person, people, selected }) => {
+  const { sex, born, died, motherName, fatherName } = person;
+
+  const mother = people.find(mom => mom.name === motherName);
+  const father = people.find(dad => dad.name === fatherName);
+
+  return (
+    <tr
+      data-cy="person"
+      key={uuid()}
```

Comment:
Using `uuid()` directly in the `key` prop can lead to unnecessary re-renders because a new UUID is generated on each render. Consider using a stable identifier from the `person` object, such as `person.id` or `person.slug`, if available.